app/views/govuk_publishing_components/components/_step_by_step_nav_header.html.erb

```diff
@@ -18,13 +20,16 @@
   if tracking_id
     tracking_options ||= ({ dimension96: tracking_id }).to_json
   end
+
+  classes = %w(gem-c-step-nav-header)
+  classes << (shared_helper.get_margin_bottom)
```

Comment:
As both classes are single strings, it would be simpler just to build the string directly. So just:
```
classes = "gem-c-step-nav-header #{shared_helper.get_margin_bottom}"
```

Reply:
Thanks, good point, always happy to save a line of code.